frontend/src/pages/Agency/Agency.js

```diff
@@ -63,17 +76,22 @@ function AgencyTable() {
   const [selected, setSelected] = useState({});
 
   useEffect(() => {
-    fetch('http://localhost:8000/agency/', { method: 'GET' })
+    fetch('http://localhost:8000/agency/table/all', { 
+      method: "GET",
+      headers: {
+        Authorization: "Bearer " + getJWT(),
+      } 
+    })
     .then(res => res.json())
     .then(data => {
       setData(data.data);
       //fill in dynamic filter options from database
-      for(let dat of data.data){
-        if(!(filters.Staff.hasOwnProperty(dat.tableContent.staff))){
+      for(let dat of data.data) {
+        if(!(filters.Staff.hasOwnProperty(dat.tableContent.staff))) {
           filters.Staff[dat.tableContent.staff] = false;
         }
-        if(dat.tableContent.dateOfInitialPartnership){
-        if(!(filters["Joined In"].hasOwnProperty(dat.tableContent.dateOfInitialPartnership.substring(6)))){
+        if(dat.tableContent.dateOfInitialPartnership) {
+        if(!(filters["Joined In"].hasOwnProperty(dat.tableContent.dateOfInitialPartnership.substring(6)))) {
           let year = dat.tableContent.dateOfInitialPartnership.substring(6);
           filters["Joined In"][year] = false;
         }
```

Comment:
Not sure if it's just hard to tell but is there a missing closing brace here? Also, why are the two if statements on 93 and 94 stacked on top of each other? It seems like they should be nested and it looks like one of the if statements is missing a closing brace. Can you confirm for me if this is correct?

Reply:
Just indented line 94 to fix the indentation. The closing brace was there but because the indentation was a bit messed up, it was hard to tell.